ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,130 @@
+import argparse
+
+
+def _process_image_for_icon(img, size: int):
+    """Обрабатывает изображение для создания иконки"""
+    from PIL import Image
+
+    # Конвертируем в RGBA если нужно
+    if img.mode != "RGBA":
+        img = img.convert("RGBA")
+
+    # Изменяем размер с сохранением качества
```

Ruff (0.13.1)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)